mnq/sns-instances-notification-system/README.md

```diff
@@ -0,0 +1,40 @@
+# SNS Instances Notification System
+
+This repository contains the source code for the this tutorial: [Creating a notification system with Scaleway SNS and Instances.](https://github.com/scaleway/docs-content/blob/int-add-mnq-tuto/tutorials/sns-instances-notification-system/index.mdx)
+
+## Requirements
+
+This example assumes that you are familiar with:
+
+- how System Notification Service work. You can find out more in the [SNS Scaleway official documentation](https://www.scaleway.com/en/docs/serverless/messaging/quickstart/)
+- how Instances work. If needed, you can visit the [Instances Quickstart documentation](https://www.scaleway.com/en/docs/compute/instances/quickstart/).
+- how to create a SSH key. If not done already, you can follow the first 6 steps of [this walkthrough](https://www.scaleway.com/en/docs/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/#how-to-upload-the-public-ssh-key-to-the-scaleway-interface).
+
+## Context
+
+This example shows you how to set up a Notification System with Terraform across Instances using Scaleway's products, through a simulated CPU monitoring example.
+
+## Setup
+
+Once you have cloned this repository you can run:
```

Comment:
**suggestion(usability):** to avoid the user having to manually copy-paste values for their SSH keys when they run Terraform, you can change this line to the following:

------------

Once you have cloned this repository, you can use Terraform to create all the infrastructure. 

To create instances, you need to provide Terraform with your public SSH key(s). You can do this via an environment variable:

```console
export TF_VAR_public_ssh_key=$(cat ~/.ssh/*.pub)
```